android/jni/Android.mk

```diff
@@ -17,10 +17,16 @@ LOCAL_MODULE := libpep_engine_sequoia_backend
 LOCAL_SRC_FILES := $(GPGBUILD)/$(TARGET_ARCH_ABI)/lib/libpep_engine_sequoia_backend.a
 include $(PREBUILT_STATIC_LIBRARY)
 
+include $(CLEAR_VARS)
+LOCAL_MODULE := jitterentropy
+LOCAL_SRC_FILES := $(GPGBUILD)/$(TARGET_ARCH_ABI)/lib/libjitterentropy.a
+include $(PREBUILT_STATIC_LIBRARY)
+
 $(warning ==== JNIADAPTER android.mk using BOTAN2 for pEpEngineSequoiaBackend)
 include $(CLEAR_VARS)
 LOCAL_MODULE := botan
 LOCAL_SRC_FILES := $(GPGBUILD)/$(TARGET_ARCH_ABI)/lib/libbotan-2.a
+LOCAL_CPPFLAGS += -O3
```

Comment:
Not sure that is needed, because botan gets configured explicitly anyways with its flags. But I guess you tested that.

Reply:
not sure if there is any difference to be honest as the times range always from 15 to 40 seconds.

Reply:
Removed.